### DISCUSSION TITLE
How to use abbreviations

### DISCUSSION THREAD

**Discussion by SomeMichael** [AnySoftKeyboard/AnySoftKeyboard#3363] [Q&A]
Hi! I tried to use abbreviations, but I can't get em working. What do I have to do in order to replace the abbreviation with my defined text? Thanks!

**Comment by alchemiker** [comment]
Hi SomeMichael, is that even a feature on this app? Never heard of such a function before...

**Reply by SomeMichael** [reply]
Yes it is. 😉
But i found meanwhile found out how  to use them. Totally forgot about this thread😄. Thanks for the hint, alchemiker!

**Reply by alchemiker** [reply]
Hey SomeMichael, can you then eventually share your wisdom of how to use Abbreviations with AnySoftKeyboard? The only feature in this direction that I found is the smiley alternative text...
Thanks in advance! : )

**Comment by SomeMichael** [comment]
Hi alchemiker!
Share my wisdom? Uh, well, I can try😄
Go to settings > language > second entry with something about dictionaries > abbreviations editor. The entries may differ a little bit, since I don't have English for my settings and I don't know how to change the settings language.
Then create your entries, for example :
eg > for example
Make sure to do this for the correct language! 
Then in the keyboard itself (again : language!) you can type eg and the keyboard will offer a selection "for example". Just tap it.

**Reply by alchemiker** [reply]
Thank you SomeMichael for sheding light on this mystery unbenownst to humankind before! 🙏 
As a novice of the ASK lore, in dignity I receive this gift of wisdom of yours and will pass it on at the wiki of this project when its time has come to enlighten the users of this app of this virtuous function.
From now on you shall not be called SomeMichael anymore, but MichaelTheGreat!
XD

**Reply by SomeMichael** [reply]
😆 thank you very much for this _great_ upgrade!

**Reply by highfalutin** [reply]
I will just add that, after adding my new abbreviations under Settings > Language > Configure correction behavior and other dictionaries > Abbreviation editor, I had to force-quit the app and open it up again before I was able to use the abbreviations.
